src/Microsoft.DotNet.Darc/src/Darc/Operations/GetDependencyFlowGraphOperation.cs

```diff
@@ -129,17 +136,18 @@ private bool IsInterestingEdge(DependencyFlowEdge edge)
         /// <returns></returns>
         private string GetEdgeStyle(DependencyFlowEdge edge)
         {
+            string color = edge.OnLongestBuildPath ? "color=red" : "";
```

Comment:
>string color = edge.OnLongestBuildPath ? "color=red" : ""; [](start = 12, length = 58)

This fails accessibility really badly.  I believe we actually have colorblind people currently using the application. This data needs to be exposed in a different way.

Reply:
I'm not sure. We're already using bold, dashes, and dots for update frequencies, which is why I ultimately went with a color. I think I could use double lines for the edges, and I wonder if I graphviz has a double border for nodes. Let me look. Do you think that would work and not be confusing?

Reply:
Double borders helps colorblindness, which is definitely a good step.  We still have a, like, blindness blindness problem, so we probably need a textual representation of this information in some fasion, but that's like a whole other feature (one we should probably do, of course, but maybe not as part of this PR). (And honestly, the graph is too busy for me personally, so I'd like a textual representation of the data anyway, so I don't have to try to grok the whole mess of information)

The "bold" probably isn't good either, that's a pretty subtle change that will almost certainly fail an accessibility screen, and shouldn't be used to communicate important information. I had no idea the bold lines in the sample graph meant something and just assumed it was a rendering artifact.  But that's in the past.

Reply:
![image](https://user-images.githubusercontent.com/9666644/72652753-ecb00100-393c-11ea-8392-552e1db7e7d3.png)

Reply:
Wow that was small yikes. I think it still works. Will update the header comment with the new graph visual.